skrub/_fuzzy_join.py

```diff
@@ -102,15 +139,14 @@ def _string_encoding(
     main_cols_clean = main[main_cols].astype(str)
     aux_cols_clean = aux[aux_cols].astype(str)
 
-    if isinstance(main_cols, list) and isinstance(aux_cols, list):
-        first_col, other_cols = main_cols[0], main_cols[1:]
-        main_cols_clean = main_cols_clean[first_col].str.cat(
-            main_cols_clean[other_cols], sep="  "
-        )
-        first_col_aux, other_cols_aux = aux_cols[0], aux_cols[1:]
-        aux_cols_clean = aux_cols_clean[first_col_aux].str.cat(
-            aux_cols_clean[other_cols_aux], sep="  "
-        )
+    first_col, other_cols = main_cols[0], main_cols[1:]
+    main_cols_clean = main_cols_clean[first_col].str.cat(
```

Comment:
I think we can improve a bit the naming here:
`main_cols_clean` -> `main`
`aux_cols_clean` -> `aux`
WDYT?

Reply:
Are they? :)
We're creating a copy since we're selecting the columns, therefore we can reuse the names safely.

Reply:
Oh, got it, it's better